julia/Arrow/Project.toml

```diff
@@ -1,9 +1,10 @@
 name = "Arrow"
 uuid = "69666777-d1a9-59fb-9406-91d4454c9d45"
 authors = ["quinnj <quinn.jacobd@gmail.com>"]
-version = "0.3.0"
+version = "1.1.0"
```

Comment:
Apache Arrow libraries are currently at 2.0.0 and are about to release 3.0.0; should this version number track that as well?

Reply:
This version number tracks the currently registered package in the Julia registry; for the Apache version, I plan on updating the instructions in the README with the `Pkg` command to install the library release version according to the apache/arrow release tag (i.e. `apache-arrow-2.0.0`); that will ensure people can install the exact, official Apache release code.